src/components/App/apiFunctions.tsx

```diff
@@ -5,7 +5,7 @@ export const cleanRestaurantData = (restaurants: rawRestaurantData[]): restauran
     { id, name, city, state, telephone, tags, website, genre, attire }
       ) => {
         return {
-            id, name, city, state, telephone, tags, website, genre: genre.split(','), attire
+            id, name, city, state, telephone, tags: tags.split(', '), website, genre: genre.split(','), attire
```

Comment:
This ended up being a goofy change, most of the tags overlap with the genres (although I think this will allow one additional restaurant to populate when searching `international`)

My idea was that data that isn't displayed should still be searchable. I don't know if this is user friendly, but from backstage I like it in theory.

I'm still pretty torn if changing the data from the API to include arrays was a good idea. It's forcing some more complicated work in the search and filter than may be necessary. Perhaps it would have been smarter to turn this data into an array before it right before it gets rendered to the DOM, but I haven't thought of a good way to identify which categories would need to be arrays (`genre`) and which wouldn't (`state`)